core/new-gui/src/app/common/service/user/workflow-persist/workflow-persist.service.ts

```diff
@@ -1,8 +1,9 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { AppSettings } from '../../../app-setting';
-import { Workflow } from '../../../type/workflow';
+import { WorkflowInfo, Workflow, parseWorkflowInfo } from '../../../type/workflow';
 import { Observable } from 'rxjs/Observable';
+import { map } from 'rxjs/operators';
 
 export const WORKFLOW_URL = 'user/dictionary/validate';
```

Comment:
not part of this PR, but just wondering why the URl is called `user/dictionary/validate`, the url doesn't match the variable name

Reply:
legacy code. don't know why. Will change in later PR when handling the dictionary.